src/components/view/view.api.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "View",
+  "category": "basic",
+  "description": "An enhanced View component",
+  "extends": ["View"],
+  "extendsLink": ["https://reactnative.dev/docs/view"],
+  "modifiers": ["margins", "paddings", "alignments", "background", "borderRadius"],
+  "example": "https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/ViewScreen.js",
+  "images": [],
```

Comment:
can be removed

Reply:
I kept images array empty so it will remind us to add images to this component